kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: omnitruck
+resources:
+#  - kubernetes/canary.yaml
```

Comment:
I think we can remove this line rather than commenting it out since the canary file doesn't exist.

Reply:
done